doc/nrf/gs_installing.rst

```diff
@@ -191,6 +191,9 @@ Install the bootstrapper for west by entering the following command:
 
          pip3 install west
 
+
+.. _west_update:
+
 You only need to do this once.
 Like any other Python package, the west bootstrapper is updated regularly.
```

Comment:
Probably want to rework the 'bootstrapper' part while you are here, since that no longer exists.

Reply:
Not sure what you mean here ... but I got rid of the word "bootstrapper", if that was all.